console/account/how-to/close-account.mdx

```diff
@@ -1,69 +1,50 @@
 
-- by deactivating your Organization, which allows you to [reactivate it](#how-to-reactivate-an-organization) at a later date
-- by permanently deleting your account and personal data.
+## Closing your account or erasing your personal data: what’s the difference
 
-<Macro id="requirements" />
+<Macro id="close-vs-erase" />
 
-- A Scaleway account logged into the [console](https://console.scaleway.com)
-- [Ownership](/identity-and-access-management/iam/concepts/#owner) of the Organization you intend to delete
+## How to close your account
 
-## How to deactivate your Organization
+Closing your account will permanently delete your Organization’s servers, resources, services, and backups. However, you keep access to the Scaleway console. 
 
-Deactivating an Organization will permanently delete your Organization's servers, resources, services, and backups. It will also deactivate the account. However, you can choose to reactivate your account at a later date, eliminating the need to recreate an account from scratch.
+<Macro id="requirements" />
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Ownership](/identity-and-access-management/iam/concepts/#owner) of the Organization you intend to delete
 
 <Message type="note">
-  There is a delay of up to 10 days between the time your Organization is closed and the time your resources are effectively deleted. We recommend you to manually delete all your resources before you close your Organization. 
+  We recommend that you delete all resources before closing your account. Otherwise, the remaining resources will be billed for usage before their manual deletion, which can take up to 10 days.
 </Message>
 
-1. Open the Organization drop-down menu at the top-right corner of the Scaleway console, and click the name of the Organization.
-2. Select the **Settings** tab.
-3. Scroll down to the **Deactivate Organization** section.
-4. Click the **Deactivate Organization** button once you have read and agreed with the warning message.
-    <Message type="note">
-      The **Deactivate Organization** button only displays if you are the [Owner](/console/account/concepts/#owner) of your [Organization](/identity-and-access-management/iam/concepts/#organization).
-    </Message>
-5. Type **Deactivate**, then click **Continue Deactivation** to confirm your decision.
-6. Select the reasons why you are deactivating your Organization.
-7. Click **Deactivate Organization**. A message appears to confirm the deactivation has been scheduled.
 
-## How to reactivate an Organization
 
-When you deactivate your Organization, you can request that it be reactivated at any time.
+1. Click the **Settings** tab from your Organization dashboard (alternatively, you can go to the **User account** page.)
```

Comment:
I can't find the option to close an account under the User Account page. Will it be added to it?